.github/workflows/build.yml

```
@@ -65,11 +65,12 @@ jobs:
       - name: "Verify no plugin issues"
         run: mvn artifact:check-buildplan --batch-mode --no-transfer-progress
 
-      - name: "Verify reproducible build"
+      - name: "Do clean install of dependencies"
         # See https://maven.apache.org/guides/mini/guide-reproducible-builds.html#how-to-test-my-maven-build-reproducibility
         run: |
           mvn clean install --batch-mode --no-transfer-progress -Dproguard.skip -DskipTests
```

Could simplify this by moving the command in the same line:
```suggestion
        run: mvn clean install --batch-mode --no-transfer-progress -Dproguard.skip -DskipTests
```

---

I've fixed this. Also for the other files I've made the same change to keep everything consistent.